src/core/github.js

```diff
@@ -5,10 +5,58 @@
  */
 
 import l10n from "core/l10n";
+import css from "deps/text!deps/balloon.css";
 import { pub } from "core/pubsubhub";
 
 export const name = "core/github";
 
+function enableFileIssuesOnSelect(issueBase){
+  const { body, head } = document;
+
+  const codeStyle = document.createElement("style");
+  codeStyle.textContent = css;
+  codeStyle.classList.add('removeOnSave');
+  head.appendChild(codeStyle);
+
+  var fileIssueLink = document.createElement('button');
+  fileIssueLink.setAttribute('data-balloon', 'file an issue related to the text selected');
+  fileIssueLink.setAttribute('data-balloon-pos', 'up');
+  fileIssueLink.setAttribute('id', 'respec-file-issue-btn');
+  fileIssueLink.classList.add('removeOnSave', 'respec-file-issue-hidden');
+  fileIssueLink.innerText = 'File Issue';
+  fileIssueLink.onclick = function () {
+    window.location.href = fileIssueLink.getAttribute('data-href');
+    fileIssueLink.classList.add('respec-file-issue-hidden');
+  };
+  body.appendChild(fileIssueLink);
+
+  body.addEventListener("mouseup", function () {
+    const selectionObj = window.getSelection();
+    const { top, right, left } = selectionObj.getRangeAt(0).getBoundingClientRect();
+    const textSelected = selectionObj.toString();
+    const newIssueURL = new URL('./new',issueBase);
+    const { searchParams } = newIssueURL;
+    var titleText, bodyText;
```

Comment:
It seems `titleText` is used only within the following `if` block. We can use `let` to declare a variable in block scope.